### PR TITLE
Add index name to refresh mapping task

### PR DESCRIPTION
**Pull request by GeChenxin**
refresh mapping add index name

When I call _cat/pending_tasks find many "refresh-mapping" tasks, but I can't find which index case refresh-mapping task. So I add the index name in refresh mapping task description.

### CODE REVIEW

**Comment by GeChenxin** [comment]
@dnhatn Can you help me to reivew this PR? Thank you!

File: server/src/main/java/org/elasticsearch/cluster/metadata/MetadataMappingService.java [hunk 1]
```diff
@@ -198,7 +198,7 @@ private boolean refreshIndexMapping(IndexService indexService, IndexMetadata.Bui
      */
     public void refreshMapping(final String index, final String indexUUID) {
         final RefreshTask refreshTask = new RefreshTask(index, indexUUID);
-        clusterService.submitStateUpdateTask("refresh-mapping",
+        clusterService.submitStateUpdateTask(String.format("refresh-mapping [%s/%s]", index, indexUUID),
```
Comment (dnhatn) [inline server/src/main/java/org/elasticsearch/cluster/metadata/MetadataMappingService.java:201]: Let's add only the index name to the task name. Also, we can just concatenate strings (i.e., `"refresh-mapping[" + index + "]"`); no need to use String.format.

**Review by dnhatn — COMMENTED** [review]
Thanks @GeChenxin. I've left a comment.

**Comment by dnhatn** [comment]
@elasticmachine update branch

**Comment by dnhatn** [comment]
@elasticmachine ok to test

**Review by dnhatn — APPROVED** [review]
LGTM

**Comment by dnhatn** [comment]
@GeChenxin Thank you for your contribution.

**Pull request merged**